src/js/core/RowManager.js

```diff
@@ -929,6 +929,11 @@ export default class RowManager extends CoreFeature{
 				}
 				
 				this.layoutRefresh(true);
+				
+				// handle resized columns
+				if(this.table.modExists("groupRows")) {
+					this.table.modules.groupRows.virtualRenderFill();
```

Comment:
Could you do this inside the [GroupRows](https://github.com/olifolkerd/tabulator/blob/d6b654c4385beefd32fdc808cc695a6577e4cd6e/src/js/modules/GroupRows/GroupRows.js) module? Just making sure that we are isolating group rows related codes within its own module.

You can probably subscribe to a `table-layout` event from the GroupRows module and see if that works. If not, please let me know, and we can figure something out.